src/triage/component/catwalk/utils.py

```diff
@@ -201,7 +201,7 @@ def retrieve_model_hash_from_id(db_engine, model_id):
 
 
 def _write_csv(file_like, db_objects, type_of_object):
-    writer = csv.writer(file_like, quoting=csv.QUOTE_MINIMAL)
+    writer = csv.writer(file_like, quoting=csv.QUOTE_MINIMAL, lineterminator='\n')
```

Comment:
I'm pretty sure this is unrelated, but I'm curious, is this fixed by ensuring that the file object that's passed to this function – when it is genuinely a _file_ – is opened with `newline=''`?

https://docs.python.org/3/library/csv.html#csv.writer

(I would think that it's not a big deal to always write with a line terminator of `\n`, but also that this can be left to the OS. If Windows wants to be dumb, fine; and, perhaps we can allow it to meet the expectations of its ecosystem.)

Reply:
When I tested this with the Triage version of a couple of months ago, if we don't specify the \n then the copy command fails to copy the csv to the expected database table format.

Reply:
Sure, I gotchya. And, if I remember correctly ... that would've been the `StringIO`-based version?

Anyway, if that's the case, I'm guessing yeah that that's still necessary 🤷‍♂️ 

Windows, man.